### PR TITLE
improve(website): api references

### LINKED ISSUE #2427: Add API reference to documentation
We have an OpenAPI document, we need a good way to render it into a page on our website.

### PR DESCRIPTION
**Pull request by mxyng**
## Summary

Basic implementation of API docs using `@stoplightio/elements`. Pulls versioned openapi spec directly from github which probably isn't the smartest solution.

## TODO

- [x] Use spec file from the repo
- [x] Dynamically route to discovered API versions
- [ ] Add right hand nav to switch between versions
- [ ] Add left hand nav for other available docs

## Checklist

- [ ] Wrote appropriate unit tests
- [ ] Considered security implications of the change
- [ ] Updated associated docs where necessary
- [ ] Updated associated configuration where necessary
- [ ] Change is backwards compatible if it needs to be (user can upgrade without manual steps?)
- [ ] Nothing sensitive logged
- [ ] Considered data migrations for smooth upgrades

## Related Issues

Resolves #2427

### CODE REVIEW

**Comment by ssoroka** [comment]
how does this look when rendered?

**Comment by mxyng** [comment]
It's far from complete but you can get a preview here https://infra-1ekpbf9fx-infrahq.vercel.app/docs/reference/api-reference/v0.14.1

**Comment by jmorganca** [comment]
Very cool start. Excited to see where this goes

File: website/styles/globals.css [hunk 1]
```diff
@@ -227,3 +227,7 @@ body,
 .token.variable {
   color: #ff6666;
 }
+
+.sl-elements {
+  --color-code: bg-black;
+}
```
Comment (hoyyeva) [inline website/styles/globals.css:233]: 💯 🤩

**Comment by mxyng** [comment]
Closing for now

**Pull request closed without merging**